EpiAware/docs/src/examples/getting_started.jl

```diff
@@ -171,7 +171,7 @@ We choose a simple observation model where infections are observed 0, 1, 2, 3 da
 obs_model = EpiAware.DelayObservations(
     fill(0.25, 4),
     time_horizon,
-    truncated(Gamma(5, 0.05 / 5), 1e-3, 1.0)
+    truncated(Normal(0, 0.1 * sqrt(pi) / sqrt(2)), 0.0, Inf)
```

Comment:
This is a bit painful to expect people to write. Maybe we need an issue for a helper? Its quite low priority though obviously.

Reply:
I agree. In the back of my head, this should be a constructor function for `DelayObservations`.

For example, we could have a kwarg like `varprior_mean = ...` and its understood that this creates a Half-Normal prior for the variance component of the model with the desire mean.

Reply:
Made this an issue.